<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="logTextEdit">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>-7</y>
      <width>511</width>
      <height>351</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>516</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionRevertFile"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="actionShowCalculator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="actionRevertFile">
   <property name="text">
    <string>Revert</string>
   </property>
   <property name="toolTip">
    <string>Revert</string>
   </property>
  </action>
  <action name="actionShowCalculator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Calculator</string>
   </property>
   <property name="toolTip">
    <string>Calculator</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
